web/session.py

```diff
@@ -309,12 +309,15 @@ def __delitem__(self, key):
             os.remove(path)
 
     def cleanup(self, timeout):
+        if not os.path.isdir(self.root):
+            return
+
         now = time.time()
         for f in os.listdir(self.root):
             path = self._get_path(f)
             atime = os.stat(path).st_atime
             if now - atime > timeout:
-                os.remove(path)
+                shutil.rmtree(path)
```

Comment:
With `shutil.rmtree(path)`, I am getting a `NotADirectoryError` during session cleanup. It looks like the sessions directory just contains files for all the sessions, and rolling back to `os.remove(path)` seems to be working.

Reply:
@freesolox Which OS are you running? Windows?